docs/manual-CN/1.overview/2.quick-start/2.FAQ.md

```diff
@@ -107,51 +92,76 @@ compact 的主要目的是通过重新组织硬盘上的文件结构，使得从
 
 多块硬盘时可以逗号分隔多个目录，每个目录对应一个 RocksDB 实例，以有更好的并发能力。参考[这里](../../3.build-develop-and-administration/3.configurations/5.storage-config.md)。
 
+## 常见错误排查
+
 ### 连接失败，报错 Connection refused
 
 ```txt
 E1121 04:49:34.563858   256 GraphClient.cpp:54] Thrift rpc call failed: AsyncSocketException: connect failed, type = Socket not open, errno = 111 (Connection refused): Connection refused
 ```
 
-1. 检查服务是否存在
-
-```bash
-$ /usr/local/nebula/scripts/nebula.service status all
-```
-
-或者
-
-```ngql
-nebula> SHOW HOSTS;
-```
-
-2. 请确保以下端口可用，且未被防火墙阻止。您可以在 `/usr/local/nebula/etc/` 的配置文件中找到相关的端口信息。
+- 请确保以下端口可用，且未被防火墙阻止。
 
 服务名称|端口号
 ---|---
 Meta 服务|45500, 11000 - 11002
 Storage 服务|44500, 12000 - 12002
 Graph 服务|3699, 13000 - 13002
 
-此外，如果您更改了端口的默认值，请重新启动服务。
-
 运行以下命令查看端口是否可用：
 
 ```bash
 $ telnet $ip $port
 ```
 
-确保集群中所有主机的端口均可用。
+如果您更改了端口的默认值（`/usr/local/nebula/etc/`），请重新启动服务。
+
+- 检查服务是否存在
+
+```bash
+$ /usr/local/nebula/scripts/nebula.service status all
+```
```

Comment:
indent

Reply:
No indent. otherwise the preview is wrong.